packages/grafana-ui/src/components/Table/TableCellBuilder.tsx

```diff
@@ -58,6 +58,8 @@ export interface ColumnStyle {
   valueMaps?: any;
   rangeMaps?: any;
 
+  align?: string;
```

Comment:
Consider typing `align` as  'left' | 'center' | 'right' | 'justify'
```suggestion
  align?: 'default' | 'left' | 'center' | 'right' | 'justify';
```

Reply:
Aggreed.